src/components/Pagination/Pagination.scss

```diff
@@ -25,6 +25,14 @@ $stacking-order: (
       .Button {
         flex: 1 0 auto;
       }
+
+      span {
+        flex: 1 0 auto;
+
+        .Button {
+          width: 100%;
+        }
```

Comment:
Getting a linting error because it's doing 3 levels of nesting. Anyone know a better way to do this?

Reply:
Because the span will get added inside the other component I think this is your best bet. Good job on explicitly adding the span in the prop, it avoids surprises.